identity/src/main/java/com/android/identity/presentationlog/PresentationLogMetadata.kt

```diff
@@ -0,0 +1,166 @@
+package com.android.identity.presentationlog
+
+import androidx.annotation.VisibleForTesting
+import co.nstant.`in`.cbor.CborBuilder
+import co.nstant.`in`.cbor.CborEncoder
+import com.android.identity.internal.Util
+import com.android.identity.util.EngagementTypeDef
+import com.android.identity.util.Timestamp
+import java.io.ByteArrayOutputStream
+
+class PresentationLogMetadata private constructor(
+    val presentationTransactionStatus: PresentationTransactionStatus,
+    val engagementType: EngagementTypeDef,
+    val error: String = "",
+    val transactionStartTime: Long,
+    val transactionEndTime: Long,
+    val sessionTranscript: ByteArray,
+    val locationLatitude: Double = 0.0,
+    val locationLongitude: Double = 0.0
+) {
+
+    // return instance data in CBOR encoded format
+    val cborDataBytes: ByteArray
```

Comment:
once a `PresentationLogMetadata` has been built with specified entries, return the CBOR-encoded bytes of all the fields.